cmd/k8s-bigip-ctlr/main.go

```diff
@@ -419,7 +446,8 @@ func main() {
 		log.Fatalf("error connecting to the client: %v", err)
 	}
 	if *manageRoutes {
-		appMgrParms.RouteClientV1, err = routeclient.New(config)
+		rclient, err := routeclient.New(config)
```

Comment:
Now that we know what the actual command line options for routes are, we should remove manageRoutes and use the routeVserverAddr as the flag to turn on route support. Also, all of the new options should be 'hidden' until we're done with the feature.